topics/metabolomics/tutorials/gcms/tutorial.md

```diff
@@ -0,0 +1,432 @@
+> {: .text-justify}
+> {% snippet faqs/galaxy/datasets_rename.md %}
+>
+{: .comment}
+
+Before the next step with msPurity package on MS/MS datas, here are some questions to be able to verify if your file is ready and if you have the same results as us. Please check these questions : 
+{: .text-justify}
+
+> ### {% icon question %} Question before MS/MS steps
+> 
+>  **1** - What are the steps of XCMS you made before your final file ?
+> > ### {% icon solution %} Solution
+> > 
+> > Here are the different steps made for our example : 
+> >  - **(Not with XCMS)** import your datas into Galaxy instance
+> >  - **MSNbase readMSData** {% icon tool %} to read our MS datas
+> >  - XCMS peakpicking with **xcms findChromPeaks (xcmsSet)** {% icon tool %} tool
+> >  - (Not with XCMS but necessary) merge my datas into one file with **xcms findChromPeaks Merger** {% icon tool %} tool
+> >  - XCMS grouping with **xcms groupChromPeaks (group)** {% icon tool %} tool
+> >  - **(Not done)** XCMS retention time correction, then grouping again with xcms adjustRtime (retcor) {% icon tool %} tool
+> >  - XCMS integration of missing peaks with **xcms fillChromPeaks (fillPeaks)** {% icon tool %} tool
+> > 
+> {: .solution}
+> <br>
+>  **2** - Concerning what we said before and the previous answer, what is the complete name of your final RData file ?
+> > ### {% icon solution %} Solution
+> > 
+> > During each step of XCMS pre-processing, the name of the file which is processing is completed by the name of the step you were doing. So, finally your file should be name `xset.merged.groupChromPeaks.fillChromPeaks.RData`. That because (as seen in previous answer) you ran a grouping and the integration after merged datas.
+> > {: .text-justify}
+> > 
+> {: .solution}
+> <br>
+> **3** - What is the size (in MB) of your final RData file ?
+> > ### {% icon solution %} Solution
+> > 
+> > To be able to see the size of a file in your history, you just have to select it. It will deploy informations about it and you can see the size of yours. For our example, the size of the final file is **1.4 MB**.
+> > {: .text-justify}
+> > 
+> {: .solution}
+{: .question}
+
+
+# Processing with metaMS part
+
+**metaMS** is a R package for MS-based metabolomics data. It can does basic peak picking and grouping using functions from **XCMS** and **CAMERA** packages. The main output of **metaMS** is a table of feature intensities in all samples which can be analysed with multivariate methods immediately. The package also offers the possibility to create in-house databases of mass spectra (including retention information) of pure chemical compounds. These databases can then be used for annotation purposes. The most important functions of this package are *runGC* and *runLC* (and each one to create databases *createSTDdbGC* and *createSTDdbLC*).
```

Comment:
```suggestion
**metaMS** is a R package for MS-based metabolomics data. It can do basic peak picking and grouping using functions from **XCMS** and **CAMERA** packages. The main output of **metaMS** is a table of feature intensities in all samples which can be analyzed with multivariate methods immediately. The package also offers the possibility to create in-house databases of mass spectra (including retention information) of pure chemical compounds. These databases can then be used for annotation purposes. The most important functions of this package are *runGC* and *runLC* (and each one to create databases *createSTDdbGC* and *createSTDdbLC*).
```